### ISSUE TITLE
Pack fails to install on Python 3.8 

### ISSUE THREAD

**Issue by bmarick** [StackStorm-Exchange/stackstorm-salt#24]
The pack fails to install on Python 3.8.
I found the solution was to install `distro` before installing the package.

While I can add that to the requirements, should we possibly switch to a newer version of the `salt` module?
By moving to a new version of the module we would loose Python2.7 support.

I am happy to submit the PR either way.
Just want feed back before I submit the PR

**Comment by nzlosh** [comment]
Thank you for opening this issue @bmarick 

Salt stopped active development on 3001 since 31 Dec 2021.  There is an [open PR](https://github.com/StackStorm-Exchange/stackstorm-salt/pull/23) to remove the pinning which is currently waiting on the CLA being signed.  I've asked for a status update on the PR and if there's no response within a week or so I'll close it.  In that case, a new PR would be most welcome.

**Comment by nzlosh** [comment]
@bmarick the author of the PR hasn't responded, would you mind raising a PR?

**Comment by bmarick** [comment]
Happy to do so!
Just trying to validate the Change in my environment

**Comment by bmarick** [comment]
@nzlosh  [PR](#25) has been submitted, #25  
I have deployed the change in my environment and validated that I can do a test.ping salt command. 
Also I have signed the CLA.

**Comment by bmarick** [comment]
@nzlosh  I see the unit tests failed on merge. Anything you need help with tracking down?

**Comment by nzlosh** [comment]
No, they were fixed here https://github.com/StackStorm-Exchange/stackstorm-salt/pull/26

**Comment by nzlosh** [comment]
Thanks again for the contribution
